examples/LoRaSimpleGateway/LoRaSimpleGateway.ino

```diff
@@ -0,0 +1,113 @@
+/*
+  LoRa Simple Gateway/Node Exemple
+
+  This code uses InvertIQ function to create a simple Gateway/Node logic.
+
+  Gateway - Sends messages with enableInvertIQ()
+          - Receives messeges with disableInvertIQ()
+
+  Node    - Sends messages with disableInvertIQ()
+          - Receives messeges with enableInvertIQ()
+
+  With this arrangement a Gateway never receive messagem from another Gateway 
+  and a Node never receive message from another Node.
+  Only Gateway to Node and vice versa.
+
+  This code receives messagens and sends a message every second.
+
+  InvertIQ function basically invert the LoRa I and Q signals.
+
+  See the Semtech datasheet, http://www.semtech.com/images/datasheet/sx1276.pdf
+  for more on InvertIQ register 0x33.
+
+  created 05 August 2018
+  by Luiz H. Cassettari
+*/
+
+#include <SPI.h>              // include libraries
+#include <LoRa.h>
+
+const long frequency = 915E6;  // LoRa Frequency
+
+const int csPin = 10;          // LoRa radio chip select
+const int resetPin = 9;        // LoRa radio reset
+const int irqPin = 2;          // change for your board; must be a hardware interrupt pin
+
+void setup() {
+  Serial.begin(115200);                   // initialize serial
```

Comment:
Let's have the default baud at 9600 like the other examples, as it's the default baud rate of the Arduino IDE.